src/config.ts

```diff
@@ -153,21 +157,26 @@ async function readConfigFile(
 }
 
 function parseConfigFile(configData: string): ConfigurationOptionsPartial {
+  let data: Record<string, unknown>
   try {
-    const data = YAML.parse(configData)
-
-    // These are the options that we support where the user can provide
-    // either a YAML list or a comma-separated string.
-    const listKeys = [
-      'allow-licenses',
-      'deny-licenses',
-      'fail-on-scopes',
-      'allow-ghsas',
-      'allow-dependencies-licenses',
-      'deny-packages',
-      'deny-groups'
-    ]
+    data = YAML.parse(configData)
+  } catch (e) {
+    throw new Error(`Can't parse YAML: ${(e as Error).message}`)
```

Comment:
Not really sure what it looks like to get an error here or what is "supposed" to happen. But in another language I would wrap the error and it [looks like that might be possible here too, if I'm reading this correctly.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error)

```suggestion
    throw new Error(`Can't parse YAML: ${(e as Error).message}`, { cause: e })
```

Although, that said, is there a reason to catch the error here at all if you're just going to re-throw it?

Reply:
I like this `cause` thingie, TIL ty! I tested it locally and it's not available in ES6 though 😢 

I introduced this after splitting nested `try` blocks, but looking at the code I don't think we're adding anything to what Node already does so I removed it in c5c5f44.